packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -169,34 +215,30 @@ export const useArbTokenBridge = (
       return error.message
     }
 
-    addTransaction({
-      type: 'deposit-l1',
-      status: 'pending',
-      value: utils.formatEther(amount),
-      txID: tx.hash,
-      assetName: 'ETH',
-      assetType: AssetType.ETH,
-      sender: walletAddress,
-      l1NetworkID,
-      l2NetworkID
-    })
-
-    const receipt = await tx.wait()
-
-    if (txLifecycle?.onTxConfirm) {
-      txLifecycle.onTxConfirm(receipt)
+    let retryableCreationTxID = ''
+    if (destinationAddress) {
+      const [l1ToL2Message] = await receipt.getL1ToL2Messages(l2.provider)
```

Comment:
We're getting data from `depositTo` with `l1ToL2Message`. Standard ETH transfers didn't change, using `ethDepositMessage`.